pixi/src/button/button.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const $btn = document.querySelector('button');
+console.log('Found $btn', $btn);
```

Comment:
nit: Here and elsewhere can we remove the console statements? 😅  Can be helpful for development but we probably don't want to leave them here in case we forget.

Reply:
Removed!